src/igvfd/schemas/treatment.json

```diff
@@ -299,7 +360,8 @@
             "type": "string",
```

Comment:
Add comment in the treatment type, providing examples of chemical, protein and environmental treeatments.

Reply:
There already exist examples for chemical & protein, so I just added one for environmental.